source/PlayerInfo.cpp

```diff
@@ -1578,18 +1592,34 @@ void PlayerInfo::UpdateMissionNPCs()
 
 
 // Accept the given job.
-void PlayerInfo::AcceptJob(const Mission &mission, UI *ui)
+void PlayerInfo::AcceptJob(const Mission &mission, UI *ui, bool acceptEscort)
```

Comment:
```suggestion
void PlayerInfo::AcceptJob(const Mission &mission, UI *ui, bool istEscortJob)
```
I think this name is better because acceptEscort seems to say we're only accepting escort jobs or something like that, whereas its just that the job selected is an escort job.